runtime/mainnet/src/config/governance.rs

```diff
@@ -80,6 +80,124 @@ impl pallet_sudo::Config for Runtime {
 	type WeightInfo = weights::pallet_sudo::WeightInfo<Runtime>;
 }
 
+/// Contains the impl to set the council members on a runtime upgrade.
+pub(crate) mod initiate_council_migration {
+	use alloc::{vec, vec::Vec};
+
+	use frame_support::traits::OnRuntimeUpgrade;
+	#[cfg(feature = "try-runtime")]
+	use sp_runtime::TryRuntimeError;
+
+	use super::*;
+
+	parameter_types! {
+		C1: AccountId = AccountId::from_ss58check("13BL7T6bTgeEdfEdZqLCKJZPN8ncyFNxxHRKFb2YMATvyfH4").expect("address is valid SS58");
```

Comment:
Perhaps more descriptive names would be nice for me.

Reply:
disclaimer: Havent checked the accounts

Reply:
Fair :) Just aimed to provide a list of councilors: C1, C2... name of each of them wasn't that important to me as the individual accounts.